libs/ballot-interpreter-vx/src/layout.ts

```diff
@@ -110,7 +110,14 @@ export async function interpretTemplate({
       ({ bounds, corners }) => ({
         bounds,
         corners,
-        targets: [...reversed(findTargets(normalized.imageData, bounds))],
+        targets: [
+          ...reversed(
+            findTargets(normalized.imageData, bounds, {
+              targetMarkPosition:
+                electionDefinition.election.ballotLayout?.targetMarkPosition,
```

Comment:
I know there's a default in `findTargets`, but now I'm wondering if the default should be set here, at a higher level, too.

Reply:
One approach would be to require the value to be provided to `findTargets`, and the `?? BallotTargetMarkPosition.Left` would be here. What do you think?